### DISCUSSION TITLE
Audience & Content Type Relation

### DISCUSSION THREAD

**Discussion by DenByDen** [VirtusLab/strapi-plugin-navigation#236] [Q&A] [answered]
Hi everyone! 

Can someone explain to me what "Audience" content type is, and what it's for? 

And is there a way to take a custom-created "slug" field from my custom content type, and use it instead of  "URL" in navigation, when the entity is been chosen?

**Comment by ltsNotMike** [comment, marked as answer]
Hello @DenByDen,

The audience field is a basic feature that provides customization functionality to your navigation items. You can enable it on the Settings Page. This will be improved soon by adding full custom types support. 

There is a feature that is in our backlog to provide the possibility for linking URLs with the field from the related content type. It should be available in version 2.1.2. At the moment, there is no way to use your `custom-slug` instead of URL.

**Reply by DenByDen** [reply]
Thanks, @ltsNotMike, if you need a hand implementing this feature with `custom-slug`, i'll glad to help
